### PR TITLE
Added Mandrill api response 

### PR DESCRIPTION
**Pull request by ioncreature**
Mandrill API for `/messages/send.json` method returns useful information about messages sending status.

### CODE REVIEW

File: src/MandrillTransport.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
-import type {Transport, SentMessageInfo} from 'nodemailer';
+import type {Transport} from 'nodemailer';
 import type MailMessage from 'nodemailer/lib/mailer/mail-message';
+import MimeNode from 'nodemailer/lib/mime-node';
```
Comment (xr0master) [inline src/MandrillTransport.ts:3]: **suggestion**: use import type and import only type you need
`import type {Envelope} from 'nodemailer/lib/mime-node';`

File: src/MandrillTransport.ts [hunk 2]
```diff
@@ -8,6 +10,20 @@ export interface Options {
   apiKey: string;
 }
 
+export type EmailStatus = {
```
Comment (xr0master) [inline src/MandrillTransport.ts:13]: **suggestion**: rename it to MandrillStatus or MandrillResponse, to better understand that this type refers specifically to the Mandrill code.

**Review by xr0master — COMMENTED** [review]
lgtm, reviewed.
Thank you.

File: src/MandrillTransport.ts (hunk 2 above)
Comment (ioncreature) [reply src/MandrillTransport.ts]: done

File: src/MandrillTransport.ts (hunk 1 above)
Comment (ioncreature) [reply src/MandrillTransport.ts]: done

**Comment by xr0master** [comment]
A new version is published. Thank you

**Pull request merged**